packages/Symfony/src/Rector/Identical/ResponseStatusCodeRector.php

```diff
@@ -0,0 +1,132 @@
+<?php declare(strict_types=1);
+
+namespace Rector\Symfony\Rector\Identical;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\BinaryOp\Identical;
+use PhpParser\NodeDumper;
+use Rector\Rector\AbstractRector;
+use Rector\RectorDefinition\CodeSample;
+use Rector\RectorDefinition\RectorDefinition;
+
+final class ResponseStatusCodeRector extends AbstractRector
+{
+    /**
+     * @var array
+     */
+    private $codeToConst = [];
+
+    public function __construct()
+    {
+        $ref = new \ReflectionClass('Symfony\\Component\\HttpFoundation\\Response');
+
+        foreach ($ref->getConstants() as $key => $value) {
+            if (strpos($key, 'HTTP_') === 0) {
```

Comment:
Instead of reflection, the Rector should work without depending on any `Response` class.

The best practise is to make specific property with map of values:

```php
private $codeToConstant = [
   200 => 'HTTP_SUCCESS'
   // ...
];
```


Then iterate it.